### PR TITLE
Initializing Content Sections

### PR DESCRIPTION
**Pull request by alexeyandreevsky**
# Issue
Need to create new content sections: updates, services, projects and team.

# Things done
- Created content sections
- Filled the sections with content
- Created first layer routing
- Stylized sections

### CODE REVIEW

File: src/app/layout.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
-import { Inter } from 'next/font/google'
+// import { Inter } from 'next/font/google'
```
Comment (pshenmic) [inline src/app/layout.js:1]: Remove unused

File: src/app/layout.js [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 import './globals.css'
 
-const inter = Inter({ subsets: ['latin'] })
+// const inter = Inter({ subsets: ['latin'] })
```
Comment (pshenmic) [inline src/app/layout.js:4]: Remove unused

File: src/app/page.js [hunk 3]
```diff
@@ -1,16 +1,88 @@
...
+    <div className='ContentBlock' {...props}>
+        { children }
+    </div>
+  )
+}
 
 export default function Home() {
+  const [content, setContent] = useState('devspace');
+  
+  const selectItemCallback = (activeItem) => {
```
Comment (pshenmic) [inline src/app/page.js:24]: Inline this function in `selectItemCallback=`

File: src/app/page.js [hunk 4]
```diff
@@ -1,16 +1,88 @@
...
+              )}
+              
+            </AnimatePresence>
+          </div>
+        </div>
+
+      </m.div>
+
+      {/* 
+        <span>pshenmic&apos;s development page</span>
```
Comment (pshenmic) [inline src/app/page.js:76]: Remove unused

File: src/app/page.module.css [hunk 5]
```diff
@@ -15,7 +15,7 @@
   max-width: var(--max-width);
   width: 100%;
   z-index: 2;
-  font-family: var(--font-mono);
+  /* font-family: var(--font-mono); */
```
Comment (pshenmic) [inline src/app/page.module.css:18]: Why is it disabled?

File: src/components/CommandLine/CommandLine.js [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+          }
+
+        } else { // earth text
+          // remove last char from string
+
+          line = line.replaceAll('<br/>', '\n')
+          line = line.substring(0, line.length - 1).replaceAll('\n', '<br/>')
+
+          if (line.length === 0) { // setting new code function
+            newDirection = true
```
Comment (pshenmic) [inline src/components/CommandLine/CommandLine.js:104]: This algo sucks, let's build up a cleaner solution on the call

File: src/components/CommandLine/CommandLine.js [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+        }
+
+      }); // setState
+
+    }, [frameTime]); // setInterval
+
+  }, [state])
+
+                     
+  return <div dangerouslySetInnerHTML={{__html: state.currentText}}></div>
```
Comment (pshenmic) [inline src/components/CommandLine/CommandLine.js:135]: Is `\n` makes you use `dangerouslySetInnerHTML` or could you explain the use case

File: src/components/Projects/ProjectsList.js [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+const variants = {
+  open: { opacity: 1, x: 0 },
+  closed: { opacity: 0, x: "-100%" }
+}
+
+export default function ProjectsList ({projects = defaultProjectsList }) {
+  const [openedItem, setOpenedItem] = useState(-1);
+
+  useEffect(() => {
+    console.log('some project was oppened. Need to animate it.')
```
Comment (pshenmic) [inline src/components/Projects/ProjectsList.js:44]: Looks like some debug stuff

**Review by pshenmic — CHANGES_REQUESTED** [review]

File: src/components/CommandLine/CommandLine.js (hunk 7 above)
Comment (alexeyandreevsky) [reply src/components/CommandLine/CommandLine.js:134]: This attribute allows the use of HTML tags such as '<br/>'.

File: src/app/page.module.css (hunk 5 above)
Comment (alexeyandreevsky) [reply src/app/page.module.css:18]: This file was created previously but is no longer used. It might be better to just delete it.

Comment (pshenmic) [reply src/app/page.module.css:18]: Yes, please delete it

File: src/app/page.module.css [hunk 9]
```diff
@@ -36,7 +36,7 @@
 
 .code {
   font-weight: 700;
-  font-family: var(--font-mono);
+  /* font-family: var(--font-mono */
```
Comment (pshenmic) [inline src/app/page.module.css:39]: Don't comment, just delete what not used

File: src/assets/styles/mixins.scss [hunk 10]
```diff
@@ -0,0 +1,138 @@
+$width-tablet: 940px;
+
+@mixin listItem {
+    display: flex;
+    background: rgba(109, 124, 255, 0.07);
+    margin-bottom: 50px;
+    // translate: .15s all;
```
Comment (pshenmic) [inline src/assets/styles/mixins.scss:7]: Remove

File: src/components/Projects/Project.js [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+import './Project.scss'
+
+function Project ({ project, closeHandler, id }) {
+
+    return (
+      <m.div 
+        className = 'Project'
+        initial={{ scale: 1.2, opacity: 0 }}
+        animate={{ scale: 1, opacity: 1 }}
+        // exit={{ scale: 0.8, opacity: 0 }}
```
Comment (pshenmic) [inline src/components/Projects/Project.js:13]: Unused

File: src/components/Services/ServicesList.js [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+
+
+function ServicesList ({servicesList = defaultServicesList}) {
+    
+    const ListItems = servicesList.map((service, id) => 
+        <m.div
+            key={'service' + id}
+            initial={{ y: 30, opacity: 0 }}
+            animate={{ y: 0, opacity: 1 }}
+            // exit={{ y: -30, opacity: 0 }}
```
Comment (pshenmic) [inline src/components/Services/ServicesList.js:35]: Unused

File: src/app/page.js [hunk 13]
```diff
@@ -1,16 +1,112 @@
...
+  const updateCintent = () => {
+    const pathArray = window.location.pathname.split('/').slice(1)
+    setContent(pathArray[0] !== '' ? pathArray[0] : defaultContent)
+  }
+
+  const [render, setRender] = useState(false);
+  useEffect(() => setRender(true), [content]);
+
+
+  const pageChangeHandler = (a,b,c) => {
```
Comment (pshenmic) [inline src/app/page.js:30]: What is this?

File: src/app/page.js [hunk 14]
```diff
@@ -1,16 +1,112 @@
...
-        <div>open source blockchain developer</div>
-        <a href={"https://github.com/pshenmic"}>github.com/pshenmic</a>
-      </div>
+const defaultContent = 'updates'
+
+export default function App({Component, pageProps}) {
+
+  const [content, setContent] = useState('updates');
+
+  const updateCintent = () => {
```
Comment (pshenmic) [inline src/app/page.js:21]: content

**Review by pshenmic — CHANGES_REQUESTED** [review]

File: src/enums/postTypesEnum.js [hunk 15]
```diff
@@ -0,0 +1,5 @@
+export const postTypesEnum = {
```
Comment (pshenmic) [inline src/enums/postTypesEnum.js:1]: PostTypeEnum

File: src/enums/postTypesEnum.js [hunk 16]
```diff
@@ -0,0 +1,5 @@
...
+    static: 0,
```
Comment (pshenmic) [inline src/enums/postTypesEnum.js:2]: Make uppercase

File: src/components/Updates/Post.js [hunk 17]
```diff
@@ -0,0 +1,62 @@
...
+                target="_blank"
+            >
+                { children }
+            </Link>
+        );
+    };
+
+    return (
+        <m.div 
+            className={"Post " + "Post--" + postType}
```
Comment (pshenmic) [inline src/components/Updates/Post.js:29]: Make a mapping

File: src/components/Projects/ProjectsList.js [hunk 18]
```diff
@@ -0,0 +1,75 @@
...
+    title: 'Dashmate',
+    description: 'Masternode setup tool that drastically eases node configuration and setup with a nice & easy CLI interface. Built on top of Node.Js and Docker.',
+    imgSrc: '/assets/img/dashmate-preview-300x300.jpg',
+    gihubLink: '',
+    projectLink: 'https://www.dashmate.org/'
+  }
+]
+
+export default function ProjectsList ({projects = defaultProjectsList }) {
+  const [openedItem, setOpenedItem] = useState(-1);
```
Comment (pshenmic) [inline src/components/Projects/ProjectsList.js:40]: Replace with null

File: src/components/ContentContainer/ContentContainer.scss
Comment (pshenmic) [inline src/components/ContentContainer/ContentContainer.scss:1]: Remove

File: src/components/ContentContainer/index.js
Comment (pshenmic) [inline src/components/ContentContainer/index.js:1]: Remove

**Review by pshenmic — CHANGES_REQUESTED** [review]

**Review by pshenmic — APPROVED** [review]
LGTM

**Pull request merged**